draft-ietf-6man-pio-pflag.xml

```diff
@@ -251,9 +251,7 @@ When a host requests a prefix via DHCPv6-PD, it MUST use the prefix length hint
 to form addresses from the delegated prefix instead of using individual addresses in the
 on-link prefix(es). Therefore, when the host requests a prefix using DHCPv6-PD, the host
 SHOULD NOT use SLAAC to obtain IPv6 addresses from PIOs with the P and A bits set.
-Similarly, unless the host processes at least one PIO with the P bit not set, the host
-SHOULD NOT request individual IPv6 addresses from DHCPv6, i.e., it SHOULD NOT include
-any IA_NA or IA_TA options in Solicit, Renew or Rebind messages.</t>
+Similarly, if all PIOs processed by the host have the P bit set, the host SHOULD NOT request individual IPv6 addresses from DHCPv6, i.e., it SHOULD NOT include any IA_NA or IA_TA options in Solicit, Renew or Rebind messages.</t>
```

Comment:
If the host does not include IA_NA in RENEWs then the addresses expire. Maybe we could say:

If all PIOs processed by the host have the P bit set, the host SHOULD NOT request individual IPv6 addresses from DHCPv6, i.e., it SHOULD NOT include any IA_NA options in SOLICIT messages. The host MAY continue to use addresses that are already configured.

Reply:
Makes sense, I'll update the text.